Automattic/wp-calypso
# Plans: add CTA buttons to plan features in `My Plan` tab for relevant Jetpack plans.

Previously, there was no CTA button for 
- `Marketing Automation` (Jetpack Premium and Pro)
- `Spam Filtering` (paid Jetpack plans)
- `Jetpack Video`  (Jetpack Premium and Pro)
feature cards in `My Plan` tab. This PR adds those with relevant redirects.

![screen shot 2018-05-03 at 23 51 07](https://user-images.githubusercontent.com/13561163/39606302-af5de11a-4f2c-11e8-8b9c-f381506ce43f.png)

## To test:
- use calypso.live or checkout this branch
- go to `plans/my-plan/:site` for a Jetpack Premium or Professional sites
- verify that all buttons are present and redirect to correct places
